R/json_test.R

```diff
@@ -105,7 +106,8 @@ json_test <- function(
   } else {
     stop("No function specified to test.")
   }
-  fnc <- getExportedValue(package, func)
+  fnc <- ifelse(class(func) == "function", func, getExportedValue(package, func))
+  func <- ifelse(class(func) == "function", deparse(match.call()$func), func)
```

Comment:
`ifelse()` might return a vector if `func` has multiple classes, a safer way might be:

```suggestion
  fnc <- ifelse(is.function(func), func, getExportedValue(package, func))
  func <- ifelse(is.function(func), deparse(match.call()$func), func)
```

This could maybe use a comment too about why there are two very similarly named variables

Reply:
good catch! thanks.